dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task250604UpdateFolderInodes.java

```diff
@@ -38,11 +32,12 @@ public boolean forceRun() {
     @Override
     public void executeUpgrade() throws DotDataException, DotRuntimeException {
 
-        APILocator.getFolderAPI().fixFolderIds();
-
+        //Running Update Assets Inconsistencies before fixing folder IDs
+        FixTasksExecutor fixtask = FixTasksExecutor.getInstance();
+        fixtask.execute(null);
```

Comment:
I don't think I would run all the fixtasks.  I think I would enstanciate a `new FixTask00090RecreateMissingFoldersInParentPath()`

and only run that. There are risks running the whole suite of FixTasks and without doing a lot of backtesting I am not sure it is a good idea.